src/components/custom-field-input-currency/custom-field-input-currency.md

```diff
@@ -0,0 +1,62 @@
+This component may be used to express currency. Provide a numeric value to the component along with a currency code, and
```

Comment:
We need a link to our list of supported currencies:

https://github.com/mavenlink/design-system/blob/master/src/components/custom-field-input-currency/currencies.js